app/views/quotes/index.html.erb

```diff
@@ -1,15 +1,20 @@
 <h1>Pithy quotes</h1>
 
 <% @quotes.each do |quote| %>
-  <section class="quote-container">
+<section class="quote-container">
     <blockquote class="quote">
-      <%= quote.text %>
-      <footer class="author">
-        <%= quote.author %>
-      </footer>
+        <%= quote.text %>
+        <footer class="author">
+            <%= quote.author %>
+        </footer>
```

Comment:
Style-nit: can you fix this indentation -- we use 2 spaces at railsCo for HTML indents and it looks like you've used 4 in a couple places. thanks!